erpnext_com/www/docs/user/manual/en/erpnext_integration/event_streaming.md

```diff
@@ -0,0 +1,120 @@
+- For 'Delete' type only the Update Type, Document Type and Document Name is logged.
+
+![Event Update Log](/docs/assets/img/setup/integrations/event-update-log-doc.png)
+
+### 3.2 Event Sync Log
+Like the Update Log, Event Sync Log logs every document synced from the Event Producer on the Event Consumer site.
+In order to view the Event Sync Log:
+
+Go to: **Home > Integrations > Event Streaming > Event Sync Log**.
+
+![Event Sync Log](/docs/assets/img/setup/integrations/event-sync-log.png)
+
+A successfully synced event generates a log document with:
+
+- **Update Type**: Create, Update or Delete
+- **Status**: Sync Status
+- **DocType**
+- **Event Producer**: The site URL from where the document was synced
+- **Document Name**
+- **Remote Document Name**: If 'Use Same Name' is unchecked
+- **Use Same Name**
+- **Data**: The document data as JSON
+
+    ![Event Sync Log](/docs/assets/img/setup/integrations/event-synced.png)
+
+A failed event generates a log doc with the above fields along with:
+
+- **Error**: The error because of which the document was not synced.
+    ![Event Synced](/docs/assets/img/setup/integrations/event-failed-error.png)
+
+- **Resync Button**: It also provides a 'Resync' button in order to resync the failed event.
+    ![Event Failed](/docs/assets/img/setup/integrations/event-failed.png)
+
+### 3.3 Dependency Syncing
+Certain Document Types have dependencies. For example, before syncing a Sales Invoice, the Item and Customer need to be present in the current site. So, Item and Customer are dependencies for Sales Invoice. Event Streaming handles this by on-demand dependency syncing. Whenever any document is to be synced, it first checks whether the document has any dependencies (Link fields, Dynamic Link fields, Child Table fields, etc.). If that dependency is not fullfilled i.e. the dependent document (eg: Item) is not present on your consumer site, it will be synced first and then the Sales Invoice will be synced.
+
+For example: Sales Invoice syncing with Item dependency
+    ![Event Dependency](/docs/assets/img/setup/integrations/event-dependency-sync.gif)
+
+### 3.4 Naming Configuration
+Check the 'Use Same Name' checkbox to let the documents have same name on both Event Producer and Event Consumer sites. If this is not checked, then the document will be created using the naming conventions of the current site.
+
+![Use Same Name Config](/docs/assets/img/setup/integrations/event-use-same-name.png)
+
+**Note**: For Document Types that have naming series, it is adviced to keep the 'Use Same Name' checkbox unchecked, to prevent naming conflicts. If this is unchecked, the Documents are created by following the naming conventions on the current site and the 'Remote Site Name' and 'Remote Document Name' custom fields are set in the synced document to store the Event Producer site URL and the document name on the remote site respectively.
```

Comment:
```suggestion
> **Note**: For Document Types that have naming series, it is advised to keep the 'Use Same Name' checkbox unchecked, to prevent naming conflicts. If this is unchecked, the Documents are created by following the naming conventions on the current site and the 'Remote Site Name' and 'Remote Document Name' custom fields are set in the synced document to store the Event Producer site URL and the document name on the remote site respectively.
```